styleguide.md

```diff
@@ -0,0 +1,77 @@
+# Guidelines
+
+Developers will be reading your documents to learn .NET core, and to use it in their regular work.
+Your goal is to create useful documentation that helps the reader on their journey. Our guidelines
+help with that. Our style guide contains four recommendations:
+- [Use a Conversational Tone](#use-a-conversational-tone)
+- [Write in 2nd Person](#write-in-2nd-person)
+- [Use Active Voice](#use-active-voice)
+- [Target a 5th Grade Reading Level](#target-a-5th-grade-reading-level)
+
+You will see examples of each of these as you read this style guide. I've written this guide
```

Comment:
Instead of "I've written this guide" should we say "We've written this guide" ? There could potentially be multiple authors of these open docs over time. We should probably include this in the guidelines :smile: